aten/src/ATen/native/native_functions.yaml

```diff
@@ -3445,8 +3445,18 @@
 
 - func: pixel_shuffle(Tensor self, int upscale_factor) -> Tensor
 
+- func: _native_pixel_shuffle(Tensor self, int upscale_factor) -> Tensor
+  dispatch:
+    CPU: pixel_shuffle_cpu
+    CompositeImplicitAutograd: math_pixel_shuffle
+
 - func: pixel_unshuffle(Tensor self, int downscale_factor) -> Tensor
 
+- func: _native_pixel_unshuffle(Tensor self, int downscale_factor) -> Tensor
+  dispatch:
+    CPU: pixel_unshuffle_cpu
+    CompositeImplicitAutograd: math_pixel_unshuffle
+
```

Comment:
Why introduce `_native_pixel_shuffle` / `_native_pixel_unshuffle` rather than just change the dispatch key entries for `pixel_shuffle` / `pixel_unshuffle` directly?

Reply:
remove `_native_pixel_shuffle` and `_native_pixel_unshuffle`